src/main/java/ch/uzh/ifi/hase/soprafs22/entity/User.java

```diff
@@ -66,6 +72,9 @@ public class User implements Serializable {
     @OneToOne
     private UserBlackCards userBlackCards;
 
+    @ElementCollection
+    private List<Gender> genderPreferences = new ArrayList<>();
+
```

Comment:
Use a Set here, also, you would use OneToMany for List.

Reply:
is set not, but keeping it as element collection